index.bs

```diff
@@ -2663,17 +2724,15 @@ To <dfn>create [=aggregatable contributions=]</dfn> given an [=attribution sourc
         1. If |aggregationKeys|[|sourceKey|] does not [=map/exist=], [=iteration/continue=].
         1. [=map/Set=] |aggregationKeys|[|sourceKey|] to |aggregationKeys|[|sourceKey|] bitwise-OR |triggerData|'s
             [=aggregatable trigger data/key piece=].
-1. Let |aggregatableValues| be |trigger|'s [=attribution trigger/aggregatable values=].
-1. Let |contributions| be a new empty [=list=].
-1. [=map/iterate|For each=] |id| → |key| of |aggregationKeys|:
-    1. If |aggregatableValues|[|id|] does not [=map/exist=], [=iteration/continue=].
-    1. Let |contribution| be a new [=aggregatable contribution=] with the items:
-        : [=aggregatable contribution/key=]
-        :: |key|
-        : [=aggregatable contribution/value=]
-        :: |aggregatableValues|[|id|]
-    1. [=list/Append=] |contribution| to |contributions|.
-1. Return |contributions|.
+1. Let |aggregatableValuesConfigurations| be |trigger|'s [=attribution trigger/aggregatable values configurations=].
+1. [=list/iterate|For each=] |aggregatableValuesConfiguration| of |aggregatableValuesConfigurations|:
+    1. If the result of running [=match an attribution source's filter data against filters and negated filters=] with
```

Comment:
This will need to be rebased due to #1153.

Reply:
Thanks for letting me know!